### PR TITLE
Render StandaloneVideo long instructions client-side and use the translations

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR:
- Switches the long instructions for StandaloneVideo levels to be rendered clientside
- Fixes up a handful of levels that had differences
- Use the translated long instructions and teacher markdown for StandaloneLevels

Elijah paired with me to write the code that led to e63e69c as well as helping me identify the levels that needed to be reviewed.

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/models/levels/standalone_video.rb [hunk 1]
```diff
@@ -60,4 +60,22 @@ def enable_scrolling?
   def self.create_from_level_builder(params, level_params)
     create!(level_params.merge(user: params[:user], game: Game.standalone_video, level_num: 'custom'))
   end
+
+  def localized_long_instructions
+    return nil unless long_instructions
+    return long_instructions if I18n.en?
+    return I18n.t(name,
+      scope: [:data, "long_instructions"],
+      default: long_instructions,
```
Comment (Hamms) [inline dashboard/app/models/levels/standalone_video.rb:69]: oh, please also add `smart: true` to the `I18n.t` call here and below. It's necessary when dealing with keys that can have periods (as is the case for level names)

**Review by Hamms — APPROVED** [review]
one nit, otherwise LGTM!

File: dashboard/app/models/levels/standalone_video.rb (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/app/models/levels/standalone_video.rb:69]: Will do. Out of curiosity, why isn't `smart: true` the default?

Comment (Hamms) [reply dashboard/app/models/levels/standalone_video.rb:69]: Just because I didn't want to risk unintentional consequences when we originally added it

**Pull request merged**